components/dashboard/package.json

```diff
@@ -48,6 +50,8 @@
     "start": "craco start",
     "build": "craco build --verbose",
     "test": "craco test",
+    "test:unit": "craco test",
```

Comment:
Consequently, I'd remove the `test:unit` target here. Or rename it to `test:interactive`/`test:watch`, maybe...?

Reply:
See reply under https://github.com/gitpod-io/gitpod/pull/7611#discussion_r786737456